### ISSUE TITLE
When I use vimade, Maximizing splited window(`<c-w>|`) failed.

### ISSUE THREAD

**Issue by Delayless** [TaDaa/vimade#49]
`CTRL-W |`	Set current window width to N (default: widest possible).
I use the shortcut to maximize the current windows in vim's multiple windows.
When I use vimade, maximizing the vertically split window fails. Normally, the width of all other windows should be 0. But there's always a window that isn't.
I've tried:
1. when the same file split different windows, maximizing windows is normal.
2. set colbufsize, rowbufsize=0, it's useless.
3. It's normal when split horizontally.

**Comment by TaDaa** [comment]
sorry for the post 2 month response -- this seems like some kind odd bug.  I think the hidden jumps to fade other windows/buffers are triggering this expansion...  Is there a similar hide mapping used for vertical space? 

I think it makes sense to change the width / height invalidation to only recompute when the area is increased which should fix this

**Comment by Delayless** [comment]
Sure, press `<C-w>_`  to maximize windows for vertical space.
The following code is used to set the windows size from my vimrc. 
```
" use 'M' to maximize and 'm' to balance windows.
nnoremap <LEADER>wM <C-W>\| <C-W>_
nnoremap <LEADER>wm <C-W>=
```

**Comment by Delayless** [comment]
I think it's fixed after pulling the latest commit ec3d78050a562232c1614a701fe1a868ba4109e9.
Thank you. It's awesome.
